client/state/sharing/publicize/publicize-actions/reducer.js

```diff
@@ -8,9 +8,12 @@ import { omit, get } from 'lodash';
  * Internal dependencies
  */
 import {
-	PUBLICIZE_SHARE_ACTIONS_REQUEST,
-	PUBLICIZE_SHARE_ACTIONS_REQUEST_SUCCESS,
-	PUBLICIZE_SHARE_ACTIONS_REQUEST_FAILURE,
+	PUBLICIZE_SHARE_ACTIONS_SCHEDULED_REQUEST,
+	PUBLICIZE_SHARE_ACTIONS_SCHEDULED_REQUEST_SUCCESS,
+	PUBLICIZE_SHARE_ACTIONS_SCHEDULED_REQUEST_FAILURE,
+	PUBLICIZE_SHARE_ACTIONS_PUBLISHED_REQUEST,
+	PUBLICIZE_SHARE_ACTIONS_PUBLISHED_REQUEST_SUCCESS,
+	PUBLICIZE_SHARE_ACTIONS_PUBLISHED_REQUEST_FAILURE,
```

Comment:
On line 27, there's a global `undefined`. According to our JS coding standards, we should not use globals and write `'undefined'` instead: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#globals